models/Sheet.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// Sheet : Represent json model Sheet
+type Sheet struct {
+	Name        string  `json:"name"`
+	Orientation string  `json:"orientation,omitempty"` // Défault : landscape
```

Comment:
only english inside public repositories

Reply:
I don't see any property to translate in english ...

Reply:
Comment "Défault"